android/src/main/java/com/amplitude/android/Amplitude.kt

```diff
@@ -56,6 +57,9 @@ open class Amplitude(
         }
         this.createIdentityContainer(identityConfiguration)
 
+        if (this.configuration.offline != null) {
```

Comment:
nit: Can you please use the constant in these checks for readability.
```suggestion
        if (this.configuration.offline != AndroidNetworkConnectivityCheckerPlugin.Disabled) {
```